infrasec/aws/naming.md

```diff
@@ -79,3 +91,54 @@ e.g.
 
 * slack-pivotal-tracker-bot-test
 * rds-log-cleaner-webapp-prod
+
+### SNS topics
+
+The main purpose of the SNS topics are for notifications to teams. The naming convention reflects that, closely followed by the AWS account type (Gov or Com) and action.
+
+*${team}-${account_type}-${action}* - is the general form
+
+Sample names:
+
+* app-com-notification
+* infra-gov-alert
+
+### VPC
+
+VPCs are unique by region but they should indicate which application and environment level they contain.
+
+[*${application}-]${environment}* - is the general form
+
+* *$application* - if needed, disambiguates between a similar purpose across applications, e.g. "webapp" vs "honeycomb"
+* *$environment* - can be used to distinguish different versions of the resource/app that occur during the development lifecycle, e.g. `dev`, `perf_test`, `staging` and `prod`uction
+
+* eec-prod
+* staging
+
+### Subnets
+
+Although they are unique by region, we also want subnets to be descriptively named according to their AZs and public/private status.
+
+*${VPC name}-${public/private}-${AZ}* - is the general form
+
+* *$public/private* - can be used to distinguish whether the subnet is a private or a public subnet
+* *$AZ* - Availability Zone (AZ) are within a region and are ordered initials (i.e. `us-east-2b`).
+
+In a VPC named "eec-prod":
+
+* eec-prod-public-us-east-1a
+* eec-prod-private-us-west-2c
+
+## Terraform modules
```

Comment:
Might be worth noting here that this is the format the registry *requires* in order for your module to be properly added to it, so this is less of a guideline and more of a rule.

Reply:
Ohhh man. We thought that might be but then I thought I found a counterexample I can't find again so thank you!